lib/bitcore-db/queries.ts

```diff
@@ -51,6 +51,7 @@ export const getBlocks = async (date: string, page = 0): Promise<Block[]> => {
 
   const blocks: Block[] = blocksResult.map(block => ({
     ...block,
+    time: block.time.getTime() / 1000,
```

Comment:
Is this being used some where right now or just adding for future use?

Reply:
The other `Block` queries specify `time`. This change makes the `Block` type consistent so callers never have to worry about it being undefined.